### PR TITLE
March: Add community_news

### PR DESCRIPTION
**Pull request by Pradhvan**
Add meetup details in README.md

Add community newsletter in community_news.md

### CODE REVIEW

File: 2020/March/community_news.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+* Visualising Data Structures and algorithms through animations https://visualgo.net/en
+
+* PEP 614 -- Relaxing Grammar Restrictions On Decorators https://www.python.org/dev/peps/pep-0614/ 
+
+* Python in GitHub Actions https://hynek.me/articles/python-github-actions/
+
+* Python in Production  https://hynek.me/articles/python-in-production/
+
+* Virtual Ansible Contributor Summit + Hackathon - Sunday, March 29, 2020 https://groups.google.com/forum/#!topic/ansible-outreach/iYFoB9g3cSE
```
Comment (raukadah) [inline 2020/March/community_news.md]: Move this under Upcoming events

File: 2020/March/community_news.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+## Upcoming events in & around Pune
+<!-- Add link to upcoming workshops, conferences (can be from other
+places around the world) -->
+
+* Virtual Ansible Contributor Summit + Hackathon - Sunday, March 29, 2020 
+https://groups.google.com/forum/#!topic/ansible-outreach/iYFoB9g3cSE
+
+## Thanks
+
+* Thanks to Vishal and Bhavin for organizing Mar PythonPune 2020 https://www.meetup.com/PythonPune/events/269419831/
```
Comment (bhavin192) [inline 2020/March/community_news.md]:
```suggestion
* Thanks to Vishal and Bhavin for organizing March PythonPune meetup 2020 https://www.meetup.com/PythonPune/events/269419831/
```

File: 2020/March/community_news.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+* Thank you AP Raj Shekhar and Kumar Vipin Yadav for delivering Talks.
```
Comment (bhavin192) [inline 2020/March/community_news.md]:
```suggestion
* Thank you A.P. Rajshekhar and Kumar Vipin Yadav for delivering talks this month.
```

File: 2020/March/community_news.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+* Click 7.1 Released by Pallets https://palletsprojects.com/blog/click-7-1-released/
+
+## Upcoming events in & around Pune
+<!-- Add link to upcoming workshops, conferences (can be from other
+places around the world) -->
+
+* Virtual Ansible Contributor Summit + Hackathon - Sunday, March 29, 2020 
+https://groups.google.com/forum/#!topic/ansible-outreach/iYFoB9g3cSE
+
```
Comment (bhavin192) [inline 2020/March/community_news.md]: Can you please add this 
```suggestion
* First ever remote Cloud Native Rejekts - Wednesday, April 1, 2020
https://virtual.rejekts.io/
```

Comment (bhavin192) [reply 2020/March/community_news.md]: Okay, it's already merged.

**Pull request merged**